wls-admin-service/src/main/java/de/muenchen/oss/wahllokalsystem/adminservice/client/DummyClientImpl.java

```diff
@@ -58,4 +61,22 @@ public void putWahltermindaten(String wahltagID) {
     public void deleteWahltermindaten(String wahltagID) {
         log.info("dummy client deleteWahltermindaten({}) called instead of basisdaten-service", wahltagID);
     }
+
+    @Override
+    public String generateAndExportWahllokalBenutzer(String wahltagID, List<WahllokalBenutzerModel> wahllokalUserInfoModel) {
+        return "ftpprs-1503\r\n" + "c94m3c-0365\r\n" + "v7jnkr-2161\r\n" + "evc77k-2062\r\n" + "sae447-1327\r\n"
+                + "xh3mv2-0508\r\n" + "rkqdt2-2570\r\n" + "a3c9hm-1369\r\n" + "jnhm44-0566";
```

Comment:
kann man hier bei den namen ersichtlich machen, dass es dummy daten sind? also zB so:
```suggestion
        return "dummy:ftpprs-1503\r\n" + "dummy:c94m3c-0365\r\n" + "dummy:v7jnkr-2161\r\n" + "dummy:evc77k-2062\r\n" + "sae447-1327\r\n"
                + "dummy:xh3mv2-0508\r\n" + "dummy:rkqdt2-2570\r\n" + "dummy:a3c9hm-1369\r\n" + "dummy:jnhm44-0566";
```